docs/translations/translations.json

```diff
@@ -114,7 +114,7 @@
     "/utils/no-ssr": "No SSR",
     "/utils/click-away-listener": "Click Away Listener",
     "/utils/box": "Box",
-    "/demos": "Component Demos",
+    "/demos": "Components",
```

Comment:
Should we fix the URLs and add netlify redirects? (And update any links in the docs...)

Reply:
We could batch the URLs changes at some point, I was thinking of doing:
- /css-in-js -> /styles
- /style/x -> /components/x
- /demos -> /components

I will have an SEO impact.